### DISCUSSION TITLE
Does vAmiga support network connectivity?

### DISCUSSION THREAD

**Discussion by docbobo** [dirkwhoffmann/vAmiga#725] [Q&A] [answered]
Was wondering if vAmiga supports TCP connectivity. Couldn’t find a word in the documentation, so my guess is „no“..

**Comment by dirkwhoffmann** [comment, marked as answer]
Your guess is right. vAmiga does not support TCP connectivity. The focus of the emulator is to emulate the early models (A500, A1000, A2000).

**Comment by docbobo** [comment]
Well, as far as I know,  there were/are network cards at least for the A2000. So it would still fit  the purpose, I  guess 😉

**Reply by nicolasbauw** [reply]
I'm using a PCMCIA card with my A600, for example ;-) That was my main way for transferring files, before I had a Gotek mounted as an external drive.
